src/views/tutorial-view/components/next-previous/components/directional-link-box/directional-link-box.module.css

```diff
@@ -1,101 +1,36 @@
-.card {
-  border: 1px solid magenta; /** @TODO style these to spec, remove any prev imp styles necessary - https://www.figma.com/file/VD7ahvXuXWJApeGnhbW4hv/Dev-Portal?node-id=6298%3A73041 */
-  position: relative;
-  display: block;
-  background: var(--white);
-  border-radius: 5px;
-  margin: 0;
-  padding: 20px;
-}
-
-/* visually hidden */
-.hidden {
-  clip: rect(0 0 0 0);
-  clip-path: inset(50%);
-  height: 1px;
-  overflow: hidden;
-  position: absolute;
-  white-space: nowrap;
-  width: 1px;
-}
-
 .linkbox {
-  composes: card;
   display: flex;
+  flex-wrap: wrap;
   align-items: center;
-  width: 100%;
-  background-color: var(--white);
-  color: var(--black);
-  box-shadow: var(--token-surface-mid-box-shadow);
-  transition: box-shadow 0.3s;
-
-  &[data-direction='previous'] {
-    text-align: left;
-    justify-content: flex-start;
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
-    }
-  }
-
-  &[data-direction='next'],
-  &[data-direction='final'] {
-    text-align: right;
-    justify-content: flex-end;
-
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
+  text-align: center;
+  justify-content: center;
+  height: 100%;
+
+  @media (--medium-up) {
```

Comment:
Can you update this to use one of the `--dev-dot-` prefxied `@custom-media` in [`src/pages/style.css`](https://github.com/hashicorp/dev-portal/blob/main/src/pages/style.css)? If you need a new one all good to create it there! 😊

Reply:
✅ Great call, updated!